sdk/src/manifest.rs

```diff
@@ -451,6 +451,14 @@ impl Manifest {
         self.signature_info.to_owned().and_then(|sig| sig.time)
     }
 
+    /// Returns an iterator over [`ResourceRef`][ResourceRef]s.
+    pub fn iter_resources(&self) -> impl Iterator<Item = ResourceRef> + '_ {
```

Comment:
could the name here just be "resources"

Reply:
There already exists a method named `Manifest::resources` which returns a `ResourceStore`. I think the intention with the unstable API is to phase out use of resource stores in favor of methods like  `Reader::resource_to_stream`? Resource store also has a method named resources already, so we can't move it there without a breaking change.

Reply:
Ok